app/models/purchase.rb

```diff
@@ -39,7 +39,6 @@ class Purchase < ApplicationRecord
   }
 
   before_create :combine_duplicates
-  before_destroy :remove_inventory
```

Comment:
@armahillo I can't remember off the top of my head, did we move this into the controller?